seacrowd/sea_datasets/yunshan_cup_2020/yunshan_cup_2020.py

```diff
@@ -0,0 +1,180 @@
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{DBLP:journals/corr/abs-2204-02658,
+ author    = {Yingwen Fu and
+              Jinyi Chen and
+              Nankai Lin and
+              Xixuan Huang and
+              Xin Ying Qiu and
+              Shengyi Jiang},
+ title     = {Yunshan Cup 2020: Overview of the Part-of-Speech Tagging Task for
+              Low-resourced Languages},
+ journal   = {CoRR},
+ volume    = {abs/2204.02658},
+ year      = {2022},
+ url       = {https://doi.org/10.48550/arXiv.2204.02658},
+ doi       = {10.48550/arXiv.2204.02658},
+ biburl    = {https://dblp.org/rec/journals/corr/abs-2204-02658.bib},
+ bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DATASETNAME = "yunshan_cup_2020"
+
+_DESCRIPTION = """\
+Lao POS dataset containing 11,000 sentences was released as part of Yunshan-Cup-2020 evaluation track.
+"""
+
+_HOMEPAGE = "https://github.com/GKLMIP/Yunshan-Cup-2020"
+
+_LICENSE = Licenses.UNKNOWN.value  # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_URLS = {
+    "train": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/train.txt",
+    "dev": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/dev.txt",
+    "test": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/test.txt",
+}
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class YunshanCup2020(datasets.GeneratorBasedBuilder):
+    """Lao POS dataset containing 11,000 sentences was released as part of Yunshan-Cup-2020 evaluation track."""
+
+    class_labels = ["IAC", "COJ", "ONM", "PRE", "PRS", "V", "DBQ", "IBQ", "FIX", "N", "ADJ", "DMN", "IAQ", "CLF", "PRA", "DAN", "NEG", "NTR", "REL", "PVA", "TTL", "DAQ", "PRN", "ADV", "PUNCT", "CNM"]
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="yunshan_cup_2020_source",
+            version=SOURCE_VERSION,
+            description="yunshan_cup_2020 source schema",
+            schema="source",
+            subset_id="yunshan_cup_2020",
+        ),
+        SEACrowdConfig(
+            name="yunshan_cup_2020_seacrowd_seq_label",
```

Comment:
can use var `_DATASETNAME` to create config `name`
```suggestion
            name=f"{_DATASETNAME}_source",
            version=SOURCE_VERSION,
            description="yunshan_cup_2020 source schema",
            schema="source",
            subset_id="yunshan_cup_2020",
        ),
        SEACrowdConfig(
            name=f"{_DATASETNAME}_seacrowd_seq_label",
```